src/js/view/controls/controls.js

```diff
@@ -535,5 +535,6 @@ export default class Controls {
         if (model.get('autostartMuted')) {
             addClass(this.playerContainer, 'jw-flag-autostart');
         }
+        this.controlbar.el.querySelector('.jw-slider-time').setAttribute('tabindex', '0');
```

Comment:
instead of querying you can access the element directly from the controlbar
`this.controlbar.elements.time.element()`